provisioning/baremetal_pod_test.go

```diff
@@ -208,7 +208,7 @@ func TestProxyAndCAInjection(t *testing.T) {
 	}{
 		{
 			name:       "init containers have proxy and CA information",
-			containers: newMetal3InitContainers(&images, managedProvisioning().build(), &proxy),
+			containers: newMetal3InitContainers(&images, managedProvisioning().build(), &proxy, ""),
 		},
 		{
 			name:       "metal3 containers have proxy and CA information",
```

Comment:
Can we add a test where we pass in a valid API VIP and check if the container.Env for the "metal3-machine-os-downloader" contains this API VIP?